pages/assmemblies/nels_assembly.md

```diff
@@ -0,0 +1,64 @@
+---
+title: NeLS assembly 
+contributors: [Korbinian Bösl, Erik Hjerde]
+---
+
+{% include important.html content="This is an example of a Data Management tool assembly, NeLS as an infrastructure is funded and aimed for researchers in Norway and their collaborators." %}
+
+## What is the NeLS Data Management tool assembly?
+
+The [Norwegian e-Infrastructure for Life Sciences (NeLS)](https://nels.bioinfo.no/) is an infrastructure provided by [ELIXIR Norway](https://elixir.no/). NeLS provides necessary tools for Data Management and covers [Planning](planning), [Processing](processing), [Analysing](analysing) and [Sharing](sharing) Data Life Cycle stages and offers [Data Storage](storage) capacities.
+
+## Who can use the NeLS Data Management tool assembly?
+
+NeLS and the underlying infrastructure are accessible for researchers in Norway and their collaborators. Eligible researchers can apply for storage quotas and get support through the National (Norwegian) bioinformatics support desk [contact@bioinfo.no](mailto:contact@bioinfo.no).  Most of the tools in NeLS are open source and can be reused.
+
+
+
+## For what can I use the NeLS Data Management tool assembly?
```

Comment:
Maybe you could use "you" instead of "I". I think it is suggested in the style guide.